include/oneapi/dpl/internal/common_config.h

```diff
@@ -16,12 +16,18 @@
 #ifndef _ONEDPL_COMMON_CONFIG_H
 #define _ONEDPL_COMMON_CONFIG_H
 
-#if __cplusplus < 201703L
+#ifndef _MSVC_LANG
```

Comment:
I am wondering what we do with the usage of `_MSVC_LANG` in oneDPL? Should we replace it?

Reply:
```
#ifndef _MSVC_LANG
#    define _ONEDPL_cplusplus __cplusplus
#else
#    define _ONEDPL_cplusplus _MSVC_LANG
#endif
```